src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -855,114 +798,45 @@ In the `Or add link to icon` sub-section you can add a link to an icon and previ
     "h": 330
   },
   "annotations": [{
-    "x": 1300,
-    "y": 240,
-    "w": 40,
-    "h": 40
-  }]
-}
-----
-
-After uploading an icon a third sub-section will appear.
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
-    "alias": "page-edit-configure-uploaded-icon-section",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "focus": {
-    "x": 558,
-    "y": 35,
-    "w": 800,
-    "h": 570
-  },
-  "annotations": [{
-    "x": 580,
-    "y": 300,
-    "w": 760,
-    "h": 180
+    "padding": "lg",
+    "x": 585,
+    "y": 130,
+    "w": 750,
+    "h": 60
   }]
 }
 ----
 
-[NOTE]
-====
-After uploading an icon, it will be automatically selected
-====
-
-In it you can select 
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-edit-configure-selected-icon.png",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "focus": {
-    "x": 555,
-    "y": 43,
-    "w": 796,
-    "h": 575
-  }
-}
-----
-
-or unselect an icon.
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
-    "alias": "page-edit-configure-uploaded-icon-unselected",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "focus": {
-    "x": 555,
-    "y": 43,
-    "w": 796,
-    "h": 575
-  }
-}
-----
-
-If an icon is selected and btn:[Save and close] is clicked, the selected icon will be applied in the `Page icon` section.
+In the `Or add link to icon` sub-section you can add a link to an icon and preview it.
 
 [annotation,role="data-zoomable"]
 ----
 {
   "image": {
-    "rel": "compose-configuration/page-edit-configure-set-icon.png",
-    "alias": "page-edit-configure-set-icon-focus-page-content",
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-add-link",
     "w": 1920,
     "h": 1080
   },
   "view": {},
   "focus": {
-    "x": 555,
-    "y": 150,
-    "w": 1100,
-    "h": 685
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
   },
   "annotations": [{
+    "padding": "lg",
     "x": 585,
-    "y": 440,
-    "w": 110,
-    "h": 105
+    "y": 215,
+    "w": 750,
+    "h": 60
   }]
 }
 ----
 
+After uploading an icon, available icons are shown at the bottom of the modal window.
+
 After clicking on the btn:[Save] button, the icon will be shown in the sidebar next to the page name.
```

Comment:
I'd do something in the lines of:

```
After uploading an icon, all of the available icons are shown at the bottom of the modal window.
Select one (if not already selected) and click on the btn:[save] button.

The page icon will be shown in the page edit screen and the sidebar navigation next to the page name.
```

The initial version reads a bit off to me.